website/docs/tutorials/08-interacting-with-starknet/02-call.md

```diff
@@ -0,0 +1,29 @@
+# Calling contracts
+
+## Overview
+
+This cheatcode allows you to call any `@view` function without mutating the blockchain state.  
```

Comment:
Cheatcode?

Reply:
[Correct wording. Change int to hex](https://github.com/software-mansion/protostar/pull/1063/commits/990e8d986c4bd18f1378261a2b57db17f1797408)